app/models/question.rb

```diff
@@ -0,0 +1,21 @@
+class Question < ApplicationRecord
+  belongs_to :user
+
+  delegate :name, to: :user, prefix: true
+
+  validates :message, :email_from, presence: true
+  validates :email_from, format: { with: Devise.email_regexp }
+
+  before_create :pre_create
+  after_create :send_question
+
+  private
+
+  def pre_create
+    self.email_to = ENV['CONTACT_ADMINISTRATOR_EMAIL']
```

Comment:
I think `email_to` would be the same for every question record, so I would remove the column and use the ENV variable directly in the `QuestionMailer`

Reply:
Thats an option. But, we define that the email_to may change in the future. So it would be useful to save that field in the db to know where that email was sent.